vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -0,0 +1,99 @@
+module EmsRefresh::SaveInventoryContainer
+  def save_ems_container_inventory(ems, hashes, target = nil)
+    target = ems if target.nil?
+    child_keys = [:container_nodes, :container_services, :container_groups]
+    # Save and link other subsections
+    child_keys.each do |k|
+      send("save_#{k}_inventory", ems, hashes[k], target)
+    end
+
+    ems.save!
+  end
+
+  def save_container_nodes_inventory(ems, hashes, target = nil)
+    return if hashes.nil?
+    target = ems if target.nil?
+
+    ems.container_nodes(true)
+    deletes = if target.kind_of?(ExtManagementSystem)
+                ems.container_nodes.dup
+              else
+                []
+              end
+
+    save_inventory_multi(:container_nodes, ContainerNode, ems, hashes, deletes, :ems_ref)
```

Comment:
I am going to merge #1417 very soon, so this will have to change...let's coordinate.